channels/views/invites.py

```diff
@@ -0,0 +1,24 @@
+"""Channel invitations views"""
+from rest_framework import viewsets
+
+from channels.models import Channel, ChannelInvitation
+from channels.serializers.invites import ChannelInvitationSerializer
+
+
+class ChannelInvitationViewSet(viewsets.ModelViewSet):
```

Comment:
Can you add one or more tests to cover the functionality in this file?

Reply:
This viewset also needs to be protected with permissions which I missed.